util/Migrator/DbScripts/2021-11-02_00_OrganizationSponsorship.sql

```diff
@@ -120,7 +120,7 @@ CREATE PROCEDURE [dbo].[OrganizationSponsorship_Create]
     @FriendlyName NVARCHAR(256),
     @OfferedToEmail NVARCHAR(256),
     @PlanSponsorshipType TINYINT,
-    @CloudSponsor BIT NULL,
+    @CloudSponsor BIT,
```

Comment:
This is deleted from the entity, it needs to be removed here as well

Reply:
This is an old migration script. I don't think we want to retroactively change those. I accidentally did change it but I have reverted it back.

Reply:
ah thank you. You're right, we don't want changes to old migrations